.github/workflows/main.yml

```diff
@@ -65,3 +65,9 @@ jobs:
                    -G "Ninja"
           ninja
           ninja test
+
+      - name: powerloader python based tests
+        shell: bash -l -eo pipefail {0}
+        run: |
+          export TEST_MAMBA_EXE=$(pwd)/build/powerloader
+          pytest powerloader/tests/
```

Comment:
I think the `pytest ./test` would work :)

Reply:
oops, I saw that suggestion after pushing.